chore(wallet): upgrade to mui v5

samsiegart opened the pull request.

michaelfig commented on packages/dapp-wallet/ui/src/components/NavDrawer.jsx:
```diff
@@ -1,15 +1,15 @@
 const NavDrawer = () => {
   const theme = useTheme();
   const [drawerOpened, setDrawerOpened] = useState(false);
-  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
```
Did the breakpoints actually change, or is this unrelated to the upgrade?

samsiegart replied:
Yes, they changed the boundary of `breakpoints.down` to be non-inclusive, so you have to increase it by one. They have a script which handled most of these changes https://mui.com/guides/migration-v4/#run-codemods

michaelfig commented on packages/dapp-wallet/ui/src/components/NavMenu.jsx:
```diff
@@ -1,13 +1,13 @@
+import { makeStyles, useTheme } from '@mui/styles';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import { ListItemText, ListItemIcon } from '@mui/material';
```
I wonder if there's a more specific import for these `ListItem*` things than `'@mui/material'`.

samsiegart replied:
Good catch, fixed

michaelfig reviewed (approved):
Looks really good!  Just a few questions to consider... (oh and were the changes to the `color="xxx"` properties also required by the upgrade?)

samsiegart commented:
> (oh and were the changes to the `color="xxx"` properties also required by the upgrade?)

Some were required, some were just related to the migration. I fixed the chip colors now that they're supported, and also the button color now defaults to "primary" so I:
- Removed `color="primary"` from the existing ones since it was redundant
- Added `color="cancel"` to fix the cancel buttons which were relying on the old default dark gray color
